File: 03-sum-nat/SumNat.cs
// C# program to sum all numbers from 1 to n
// Created by M.V.Harish Kumar on 06-01-2024
using System;

namespace SumNat
{
    class SumNat
    {
        static int SumToN(int n)
        {
            int sum = 1;
            for (int i = 2; i <= n; i++)
            {
                sum += i;
            }
            return sum;
        }

        static void Main(string[] args)
        {
            Console.Write("Enter value for `n`: ");
            int n = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine($"The sum from 1 to {n} is {SumToN(n)}");
        }
    }
}

File: 03-sum-nat/sum_nat.php
<?php
# PHP ython program to sum numbers from 1 to n
# Created by M.V.Harish Kumar on 17-09-2024
function sumToN($n) {
    $sum = 0;
    for ($i = 0; $i <= $n; $i++) { 
        $sum += $i;
    }
    return $sum;
}

$n = +readline("Enter value for `n`: ");
$ans = sumToN($n);
echo "The sum from 1 to $n is $ans";